marklogic-data-hub-central/ui/src/pages/TilesView.test.tsx

```diff
@@ -106,8 +106,10 @@ describe('TilesView component', () => {
 
     });
 
-    test('Verify Run tile displays from toolbar', async () => {
-        const {getByLabelText, getByText, queryByText, getByTestId} = await render(<AuthoritiesContext.Provider value={ mockDevRolesService}><TilesView/></AuthoritiesContext.Provider>);
+    test('Verify Run tile displays from toolbar and can create/edit/delete with writeStep authority', async () => {
+        const authorityService = new AuthoritiesService();
+        authorityService.setAuthorities(['readStep','writeStep','runStep']);
```

Comment:
Nice. I should probably comment after am done reviewing everything top down.
With this I think we can keep the Run tests as is and you can ignore my comment earlier regarding Run.test.